mainnet.txt

```diff
@@ -3,3 +3,8 @@ cdcf936d70726dd724e0e6a8353d8e5ba5abdd20@seed2.mainnet.pokt.network:26663
 74b4322a91c4a7f3e774648d0730c1e610494691@seed3.mainnet.pokt.network:26662
 b3235089ff302c9615ba661e13e601d9d6265b15@seed4.mainnet.pokt.network:26663
 
```

Comment:
Is it ok to list six nodes equally so that the script in README.md works?

Reply:
> Is it ok to list six nodes equally so that the script in README.md works?

Done